### PR TITLE
GIX-1031: Project timeline

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Fix timeline after API breaking changes. Only show the deadline, no progress bar.

# Changes

* Remove ProgressBar from ProjectTimeline component.
* Remove swapDuration and durationTillSwapStart utils.
* Use token param in ReviewTransaction instead of token from context. We are in a project context, but using ICPs.
* Enable logo to be nullish from backend. For now added empty string. Issue to use a default logo created.

# Tests

* Remove test cases for progress bar in ProjectTimeline.
* Remove tests for utils.

### CODE REVIEW

**Comment by lmuntaner** [comment]
@peterpeterparker @mstrasinskis please review

**Review by mstrasinskis — APPROVED** [review]
LGTM, thanks

File: frontend/src/lib/components/project-detail/ProjectTimeline.svelte [hunk 1]
```diff
@@ -1,65 +1,46 @@
...
-        </span>
-        <Value>
-          {secondsToDuration(durationTillDeadline)}
-        </Value>
-      </p>
-    </ProgressBar>
-  </div>
-{/if} -->
+{#if durationTillDeadline !== undefined}
+  <p class="push-apart">
```
Comment (peterpeterparker) [inline frontend/src/lib/components/project-detail/ProjectTimeline.svelte:26]: any particular reason not using `<KeyValuePair />` here?

Comment (lmuntaner) [reply frontend/src/lib/components/project-detail/ProjectTimeline.svelte:26]: No, good point. I'll change it in another PR.

**Pull request merged**